pandas/tests/reshape/concat/test_index.py

```diff
@@ -179,16 +178,12 @@ def test_dups_index(self):
         tm.assert_frame_equal(result.iloc[10:], df)
 
         # append
-        result = df.iloc[0:8, :].append(df.iloc[8:])
+        result = concat([df.iloc[0:8, :], df.iloc[8:]])
```

Comment:
are these redundant with the tests above?  this _may_ be better thought of as a targeted test for _append.